packages/panzer/dof-mgr/src/Panzer_DOFManager_impl.hpp

```diff
@@ -805,7 +810,7 @@ DOFManager<LO,GO>::buildGlobalUnknowns_GUN(const Tpetra::MultiVector<GO,LO,GO,pa
 
     // do a prefix sum
     GO scanResult = 0;
-    Teuchos::scan<int, GO> (*getComm(), Teuchos::REDUCE_SUM, static_cast<size_t> (localsum), Teuchos::outArg (scanResult));
+    Teuchos::scan<int, GO> (*getComm(), Teuchos::REDUCE_SUM, static_cast<int> (localsum), Teuchos::outArg (scanResult));
```

Comment:
This is only helpful for removing warnings if `GO` is signed.  You really want to cast `localsum` to `GO` since that's the type of the scan.  It will get cast from `int` to `GO` here anyway.

Reply:
thanks! will change